src/Bindings/ManualBindings_World.cpp

```diff
@@ -430,31 +430,23 @@ static int tolua_cWorld_PrepareChunk(lua_State * tolua_S)
 		public cChunkCoordCallback
 	{
 	public:
-		cCallback(lua_State * a_LuaState):
-			m_LuaState(a_LuaState),
-			m_Callback(m_LuaState, 4)
+		cCallback(cLuaState & a_LuaState)
 		{
+			m_Callback.RefStack(a_LuaState, 4);
```

Comment:
Might be better to actually read this callback on line 420, with `GetStackValues`.